### ISSUE TITLE
Add new vibrations

### ISSUE THREAD

**Issue by jessegreenberg** [phetsims/john-travoltage#349]
Discussed today, we would like to have a new vibration paradigm added by tuesday to be used for demo and discussion with sub-team on tuesday 9/1. Preferably, this will be added before Friday, to be deployed to App Store Connect so that folks can download Test Flight and the app before tuesday's meeting.

The vibrations will include
 - The electron discharge vibration from the "results" paradigm, when electrons leave the body.
 - A small buzz when each electron is added to the body.
 - Continuous low frequency buzzing while charges are in the body.

@liquatics can you add to this issue with any other information about what is needed? Are we going to play with intensity at all with these? Can you comment on desired frequencies for each?

**Comment by liquatics** [comment]
This haptic design prototype will revolve around the generation and discharge of electrons. Based off of feedback, here's what I think it should look like:

**Electron generation:** Every time an electron is generated from the interaction between the rug and the foot, we should play a short haptic buzz. This buzz should coincide with the "bloop bloop" sounds the charges make when they pop into existence with the enhanced sound on. As much as possible, we should try to make the haptics align with this sound. 

This effect is similar to what we did in BASE paradigm 3, where we had the device vibrate whenever the balloon picked up any charges from the sweater. The vibration effect we used in BASE paradigm 3 should work well for this, too.

**Electrons on Body:** We are retaining case 2 of JT paradigm 4, where once the body is holding onto electrons, there should be a vibration associated with this "charged" state. However, let's tweak the vibration for the body and make it less aggressive. 

For JT's charged body, we should have a low intensity vibration that is on for a short time (maybe 100-250ms?) and off for 1s in between buzzes. (The pattern will essentially look like this [1000 (off), 100 (on), 1000, 100] ). The idea is to give the user some low intensity, non-intrusive feedback to let them know the body has charge. So if this pattern doesn't seem to accomplish that while you're implementing it, feel free to tweak it.

**Electron Discharge:** Let's port over verbatim what we have set up in paradigm 4 case 1. This is the pulsing pattern that is associated with the electrons leaving JT's finger when close to the doorknob.

@jessegreenberg happy to discuss further if these descriptions are not sufficient!
